Sources/SwiftDocC/Model/Rendering/Tutorial/References/DownloadReference.swift

```diff
@@ -39,7 +44,7 @@ public struct DownloadReference: RenderReference, URLReference {
     public init(identifier: RenderReferenceIdentifier, renderURL url: URL, sha512Checksum: String?) {
         self.identifier = identifier
         self.url = url
-        self.sha512Checksum = sha512Checksum
+        self.checksum = sha512Checksum
     }
     
     enum CodingKeys: String, CodingKey {
```

Comment:
FWIW `CodingKeys.sha512Checksum` could get renamed as well without client impact while we're at it.